frontend/pages/settings.tsx

```diff
@@ -97,4 +106,19 @@ const Settings = ({ userInfo, authenticated }: SettingsProps) => {
   )
 }
 
+type BulkResp = {
+  submissions: Array<ApplicationSubmission>
+}
+
+Settings.getInitialProps = async (ctx: NextPageContext) => {
+  const data: BulkResp = (await doBulkLookup(
+    ['submissions', '/submissions/?format=json'],
```

Comment:
cc @rm03

Reply:
Looks like the `?format=json` is not needed as `doBulkLookup()` automatically appends this to the end of any API call. Also, why are we calling `doBulkLookup()` here instead of `doApiRequest()`?